PowerPointLabs/PowerPointLabs/ActionFramework/ShortcutsLab/Hide/HideShapeActionHandler.cs

```diff
@@ -9,7 +9,13 @@ class HideShapeActionHandler : ActionHandler
     {
         protected override void ExecuteAction(string ribbonId)
```

Comment:
HideShapeHandlers are in the folder "Hide", should the folder be renamed to "HideShape"? Similarly to "TextCollection.HideSelectedShapeTag"